doc/version_history.rst

```diff
@@ -6,6 +6,25 @@
 Version History
 ###############
 
+v1.13.2
+-------
+
+* `Model`: when enabling the model, call the alarm callback for all alarms, even those in nominal state.
+  Most alarms will usually be in nominal state.
+* `WatcherCsc`:
+
+    * When enabling the CSC: print alarm events for all alarms, even those in nominal state.
```

Comment:
Nitpicking but in line 12 you separate the `when` part of the sentence with a `,` and in this line with a `:`. Please choose one and stick to that.